### PR TITLE
feat(documentation): Document the available tasks

### PR DESCRIPTION
**Pull request by RaduW**
Add explanations about test configuration and available tasks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,3 +92,119 @@ example below (that starts 4 users, with a spawn rate of 2 per second and runs f
     .venv/bin/locust -f kafka_consumers_locustfile.py --no-web -u 4 -r 2 --run-time 20s --stop-timeout 10
 
 Please consult the locust documentation for details: https://docs.locust.io/en/1.4.3/running-locust-without-web-ui.html
+
+
+### Test Configuration
+
+Locust uses a top level file, a python file to configure the tests.
+
+`ingest-load-tester` which uses locust has adds the facility to configure tests by using yaml files. Under the hood `ingest-load-tester`
```
Comment (tonyo) [inline README.md]:
```suggestion
`ingest-load-tester` which uses locust adds the facility to configure tests by using yaml files. Under the hood `ingest-load-tester` 
```

File: README.md [hunk 2]
```diff
@@ -92,3 +92,119 @@ example below (that starts 4 users, with a spawn rate of 2 per second and runs f
...
+* duration time delta range
+* number of users
+* relative weights between the following session outcomes:
+  * ok
+  * existed
+  * errored
+  * crashed
+  * abnormal termination
+
+## event generator
```
Comment (tonyo) [inline README.md]: By "event" we actually mean "error", right? I think in Sentry's terminology sessions and transactions are also events.

File: README.md [hunk 3]
```diff
@@ -92,3 +92,119 @@ example below (that starts 4 users, with a spawn rate of 2 per second and runs f
     .venv/bin/locust -f kafka_consumers_locustfile.py --no-web -u 4 -r 2 --run-time 20s --stop-timeout 10
 
 Please consult the locust documentation for details: https://docs.locust.io/en/1.4.3/running-locust-without-web-ui.html
+
+
+### Test Configuration
+
+Locust uses a top level file, a python file to configure the tests.
```
Comment (tonyo) [inline README.md]: You're talking about one specific file here, or..? If yes, can we give its name/path here?

File: README.md [hunk 4]
```diff
@@ -92,3 +92,119 @@ example below (that starts 4 users, with a spawn rate of 2 per second and runs f
...
+
+`ingest-load-tester` which uses locust has adds the facility to configure tests by using yaml files. Under the hood `ingest-load-tester` 
+creates locust tests classes that are derived from `ConfigurableUser` class which in turn is derived from the locust `UserClass`.
+
+The `ConfigurableUser` class adds functionality that allows the tests to be configured from a yaml file.
+
+Two locust files are provided, and others can be easily added. The files have the following structure:
```
Comment (tonyo) [inline README.md]: Can we provide links to those locust files?

File: README.md (hunk 2 above)
Comment (RaduW) [reply README.md]: yes, it is true but that's how they are called all over the code, not sure if we should change the name.

File: README.md (hunk 3 above)
Comment (RaduW) [reply README.md]: Locust gets as a parameter a `locustfile` containing `User` classes, which it discovers, instantiates and uses them for the test.
If I want to run the kafka test I would call:

```
locust kafka_consumers_locustfile.py
```

If I want to use the simple locust file to send sessions I would call:

```
locust simple_locustfile.py
```

File: README.md [hunk 5]
```diff
@@ -92,3 +97,119 @@ example below (that starts 4 users, with a spawn rate of 2 per second and runs f
...
 
 Please consult the locust documentation for details: https://docs.locust.io/en/1.4.3/running-locust-without-web-ui.html
+
+
+### Test Configuration
+
+Locust uses a top level file, a python file to configure the tests.
+
+`ingest-load-tester` which uses locust adds the facility to configure tests by using yaml files. Under the hood `ingest-load-tester` 
+creates locust tests classes that are derived from `ConfigurableUser` class which in turn is derived from the locust `UserClass`.
```
Comment (tonyo) [inline README.md]:
```suggestion
Locust uses a Python file as a load testing entrypoint.

`ingest-load-tester` uses Locust internally and adds the facility to configure tests by using yaml files. Under the hood `ingest-load-tester` 
creates locust tests classes that are derived from `ConfigurableUser` class which in turn is derived from the locust `UserClass`.
```

**Review by tonyo — APPROVED** [review]

**Pull request merged**